client-src/src/pages/counselor/inq-list/inq-list.html

```diff
@@ -92,35 +92,35 @@
   <ion-card *ngFor = "let inq of (sortedInquiries?sortedInquiries:inquiries); let i = index;">
     <ion-grid>
       <ion-row>
-        <ion-col col-1 class="vertical-center-contents">
-          <p>{{inq.id}}.</p>
+        <ion-col col-1 class="absolute-center-contents">
+          <p>{{inq?.id}}.</p>
         </ion-col>
-        <ion-col col-1 class="vertical-center-contents">
-          <div *ngIf = "inq.caseIndex">
-            <div *ngIf = "inq.caseIndex.index == 3">
+        <ion-col col-1 class="absolute-center-contents">
+          <div *ngIf = "inq.followUps[0]">
+            <div *ngIf = "inq.followUps[0].caseIndex == 'Likely'">
               <ion-icon name="flash" class="case-index-3"></ion-icon>
```

Comment:
Reduce the size of case index column as low as possible as it looks a little empty space when there is not entry